portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,193 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Yichen Yao</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+    <!-- Header -->
+    <div class = "header"> Yichen Yao</div>
+    <hr align="center" width="50%">
+
+    <!-- Snap Scroll Start -->
+    <article class="scroller">
+      <!-- Introduction -->
+      <section>
+        <div class="row">
+            <div class="column">
```

Comment:
nit: Unify the indent to be 2 spaces.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
done.